particle_structs/test/test_migrate.cpp

```diff
@@ -44,7 +44,7 @@ int migrateSendRight(const char* name, PS* structure) {
                pid, local_rank);
         failures(0) = 1;
       }
-      if (e != 0 && rank != local_rank) {
+      if (e != num_elems - 1 && rank != local_rank) {
```

Comment:
Yes that is correct, since the gids were circular it was sending on `e  == num_elems-1` and receiving on `e == 0`. But now since the gids are linear it sends and receives on `e == num_elems-1`